book/2.json

```diff
@@ -1,42 +1,42 @@
 [
     {"text":""},
     {"text":"     More Options","color":"dark_gray"},
-    {"text":"\nMarkers ","color":"dark_gray"},
+    {"text":"\nMarkers ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"Sends a message in chat\nwhenever the set amount of\ntime has passed"}},
```

Comment:
Perhaps describe why this is useful. For example, it can help players know when day is and it can help video creators know when to cut for separate episodes.